WalletWasabi/Affiliation/Models/CoinjoinRequest/Body.cs

```diff
@@ -0,0 +1,12 @@
+using System.Collections.Generic;
+
+namespace WalletWasabi.Affiliation.Models.CoinjoinRequest;
+
+public record Body(
+	IEnumerable<Input> Inputs,
+	IEnumerable<Output> Outputs,
+	long Slip44CoinType,
+	decimal FeeRate,
+	long NoFeeThreshold,
+	long MinRegistrableAmount,
+	long Timestamp);
```

Comment:
Ok, we have to fix this, that's clear. This property was _decorated_ as you suggest but that was redundant because the `AffiliationFeeRateJsonConverter` is a catch-all converter for `decimals` so, while we don't have any other decimal it will work. 

Now, the real solution (i am not there yet) it to use `FeeRate`  type for the `FeeRate` property and then make sure it is serialized as expected by the Affiliation Server. So, yes, this is basically what you say except that the solution is not a decoration but a change in the type.

Reply:
Done. I created a new type of this fee.